tool-plugins/intellij/getting-started/plugin-features/README.md

```diff
@@ -95,20 +95,23 @@ Three types of Ballerina file templates are available.
 
 ![alt text](images/FileTemplates.gif)
 
-## Live Templates
+## Live Templates and Code Snippets
 
-Live templates contains boilerplate codes and lets the user to enter them easily. All available live templates can be viewed/changed at **Settings -> Editor -> Live Templates -> Ballerina**.
+Live templates / code snippets contain boilerplate codes and lets the user to enter them easily. All available live templates can be viewed/changed at **Settings -> Editor -> Live Templates -> Ballerina**.
```

Comment:
Change to "Live templates/code snippets contain boilerplate code and allows you to enter them easily."

Reply:
Changed